boards/arm/grim_reefer/grim_reefer.dts

```diff
@@ -9,42 +9,59 @@
 #include <st/f4/stm32f446r(c-e)tx-pinctrl.dtsi>
 #include <zephyr/dt-bindings/input/input-event-codes.h>
 
+
+&sys_jtrst_pb4 {
```

Comment:
Just curious what this is

Reply:
Oops - It's a leftover from trying to debug a pin that defaulted to a jtag pin rather than GPIO. No longer needed.